src/utils/index.js

```diff
@@ -34,3 +34,24 @@ export const escapeCssIdentifier = (ident) => {
 export const toMs = (s) => {
 	return Number(s.slice(0, -1).replace(',', '.')) * 1000;
 };
+
+// Fill versions to exactly 3 decimals
+export const normalizeVersion = (version) => {
+	return String(version)
+		.split('.')
+		.concat([0, 0, 0])
+		.slice(0, 3)
+		.join('.');
+};
+
+// Numerically compare version strings after normalizing them
+export const compareVersion = (a, b) => {
+	a = normalizeVersion(a);
+	b = normalizeVersion(b);
+	return a.localeCompare(b, undefined, { numeric: true });
+};
+
+// Check if the required version is installed
```

Comment:
Hmm, not completely convinced if this will be enough in all scenarios. I would have imagined three methods: `versionHigherThen`, `versionLowerThen`, `versionEquals` or the like.

Reply:
(wrong line selected, I mean the `checkVersion` method)

Reply:
Like what scenarios for example? I was assuming a greater-than-or-equals check is enough for a feature check but maybe that's too simple?

Reply:
Basically it's just the API that confuses me. `checkVersion` is not clear enough, to me it reads like it would only return `true` for an exact version match. 

Another question, why are you proposing to have `swup.version` as the first argument? Are you planning to use this function to check for plugin versions as well sometime in the future?

Reply:
How about we remove the `checkVersion` method and only keep `compareVersion`? That function is much simpler for me to understand (predict)

Reply:
Anyways, none of that is really that important. Don't want to slow us down. Ready to merge from my side, if the above version import works reliably.

This review process needs to be learned to do well, as well. It's a bit too tempting to voice an opinion on every last thing 😅🤐

Reply:
Let's find a better name then. I think it's useful to add that abstraction, the compareVersion is a bit unwieldy since it requires wrangling with -1, 0 and 1.

Reply:
Yeah the idea was to supply a utility function that's useful in all contexts, so we can't hardcode the swup version.

Reply:
`versionEqualOrHigher` ? Longer but clearer IMO

Reply:
Just looked up how the semver package calls that. It's either `satisfies` for random strings or `minVersion` for specific versions. I like `minVersion` 🤠

Reply:
Great, `minVersion` it is! Another question: How would consumers (or plugins) be able to import these functions from `utils`? My first guess was that we would need to make the function available as an instance method of `Swup`:

```js
import { minVersion } from './utils/index.js';
export default class Swup {
    // ...
    this.minVersion = minVersion;
    // ...
}
```
But probably you had something more elegant in mind?

Reply:
It should work like in the example above, by importing from swup's utils export:  

```js
import { minVersion } from 'swup/utils'
```

The advantage of **not** putting it on the swup instance is that we don't increase swup's bundle size. Only if any plugins need to check for a specific version, they can import the util and bundle it themselves. Util-on-demand 🤠

Reply:
Sounds great!